frontend/src/lib/SegmentMetricsManager.ts

```diff
@@ -211,7 +212,23 @@ export class SegmentMetricsManager implements MetricsManager {
 
   // eslint-disable-next-line class-methods-use-this
   private track(evName: string, data: Record<string, unknown>): void {
-    analytics.track(evName, data)
+    let parser = new UAParser("user-agent")
+    let parsed_ua = parser.getResult()
+
+    analytics.track(evName, data, {
+      context: {
+        ip: "0.0.0.0",
+        page: {
+          title: "",
+        },
+        browser: parsed_ua.browser.name,
+        browser_version: parsed_ua.browser.version,
+        os: parsed_ua.os.name,
+        os_version: parsed_ua.os.version,
```

Comment:
nit: I think most (all?) of the other properties use camelCase instead of snake_case here.

Reply:
Switched to camelCase